ui/mainwindow.cpp

```diff
@@ -313,6 +313,14 @@ MainWindow::MainWindow(QWidget *parent)
     connect(UiIntegration::instance(), SIGNAL(navigateToCode(QUrl,int,int)), this,
             SLOT(navigateToCode(QUrl,int,int)));
 
+    const bool developerModeEnabled = !QString::fromLocal8Bit(qgetenv("GAMMARAY_DEVELOPERMODE")).isEmpty();
```

Comment:
Nice, @chsterz we probably want to use that for the remote view FPS option too. Same for the message statistics view.